pandas/core/arrays/numpy_.py

```diff
@@ -347,13 +348,17 @@ def min(self, skipna: bool = True, **kwargs) -> Scalar:
         )
         return result
 
+    amin = min
+
     def max(self, skipna: bool = True, **kwargs) -> Scalar:
         nv.validate_max((), kwargs)
         result = masked_reductions.max(
             values=self.to_numpy(), mask=self.isna(), skipna=skipna
         )
         return result
 
+    amax = max
```

Comment:
could go on NDArrayBackedExtensionArrayMixin?

Reply:
so to keep the changes minimal have used `ea_func = getattr(type(self), func.__name__, None)` in `__array_function__` to start.

but being explicit and adding `@implements(np.amin)` to `def min()` may be a better solution. (may also want separate _HANDLED_FUNCTIONS dicts for each array)

Reply:
makes sense, thanks